### ISSUE TITLE
Error in MMM Quickstart, mmm for DelayedSaturatedMMM() specified but model.fit used

### ISSUE THREAD

**Issue by nialloulton** [pymc-labs/pymc-marketing#432]
```

mmm = DelayedSaturatedMMM()

X = data.drop('y',axis=1)
y = data['y']
model.fit(X,y)
model.plot_components_contributions();
```

**Comment by nialloulton** [comment]
https://www.pymc-marketing.io/en/stable/

**Comment by twiecki** [comment]
Good catch, want to do a PR?

**Comment by ricardoV94** [comment]
Closed via #470
